lib/matplotlib/cbook/__init__.py

```diff
@@ -138,20 +138,14 @@ class CallbackRegistry:
     Parameters
     ----------
     exception_handler : callable, optional
-       If provided must have signature ::
-
-          def handler(exc: Exception) -> None:
-
-       If not None this function will be called with any `Exception`
-       subclass raised by the callbacks in `CallbackRegistry.process`.
-       The handler may either consume the exception or re-raise.
-
-       The callable must be pickle-able.
-
-       The default handler is ::
-
-          def h(exc):
-              traceback.print_exc()
+       The *exception_handler* must be a function that takes an `Exception`
+       as single parameter, or None.  If not None, it gets called with any
```

Comment:
I was a bit confused whether it was the input, or the parameter of the callable could be `None`.
```suggestion
       If not None, the *exception_handler* must be a function that takes an `Exception`
       as single parameter.  It gets called with any
```

Reply:
yup